nusmeft.warsaw.basis.json

```diff
@@ -0,0 +1,7040 @@
+{
+  "eft": "nuSMEFT",
+  "basis": "Warsaw",
+  "metadata": {
+    "description": "Warsaw basis augmented by Sterile neutrinos. . At variance with their definition, the Wilson coefficients are defined to be dimensionful, such that $\\mathcal L=\\sum _i C_i O_i$. The set of redundant operators coincides with the choice of DSixTools (arXiv:1704.04504). The weak basis for the fermion fields is chosen such that the running dimension-6 mass matrices of charged leptons and down-type quarks are diagonal at the scale where the coefficient values are specified, while up-type quark singlet field is rotated to diagonalise the running dimension-6 up-type quark mass matrix \"from the right\".\n"

```

Comment:
There is a superfluous period after the first sentence.

The second sentence seems to be copied from the Warsaw basis JSON file, which quotes arXiv:1008.4884v3, and "their definition" refers to this reference. Since the reference is absent from the JSON file of the present basis, it might be good to change the text and not write about "their definition".